sga/context_processors.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import User
+
+from sga.backend.constants import SGA_DATETIME_FORMAT, EPOCH_FORMAT, Roles
+
+
+def logged_in_user(request):
+    # TODO: Add auth
+    try:
+        return {
+            "user_name": request.user.get_full_name(),
+            "role": request.role,
+            "Roles": Roles
+        }
```

Comment:
I personally dislike seeing too much logic in a `try/except`: in this case it is probably okay-sh but try to avoid it